lametro/events.py

```diff
@@ -252,8 +280,20 @@ def partner_search_string(self):
 class LAMetroWebEvent(dict):
     '''
     This class is for adding methods to the web event dict
-    to facilitate labeling audio appropriately.
+    to facilitate labeling and sourcing audio appropriately.
     '''
     @property
     def has_audio(self):
         return self['Audio'] != 'Not\xa0available'
+
+    @property
+    def has_detail_url(self):
```

Comment:
This seems to be an interesting thing we are evolving here. Would you open an issue on python-legistar to discuss if we want to do pursue this line.

Reply:
agreed. will do!